src/models/goal.js

```diff
@@ -167,7 +174,7 @@ export default (sequelize, DataTypes) => {
       allowNull: true,
     },
     createdVia: {
```

Comment:
Do we need column def for 'mapsToParentGoalId' here?

Reply:
line 111